src/components/buttons/base-button-wrapper/base-button-wrapper.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spacings from '../../../materials/spacings';
+import styles from './base-button-wrapper.mod.css';
+
+// This holds the logic of the `button` element, which wraps a `span` (defined as `BaseButtonContent`) with styles
+class BaseButtonWrapper extends React.Component {
+  static displayName = 'BaseButtonWrapper';
+
+  static propTypes = {
+    id: PropTypes.string,
+    name: PropTypes.string,
+    ref: PropTypes.string,
+    children: PropTypes.node,
+    ariaLabel: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+    dataAttr: PropTypes.object,
+    isDisabled: PropTypes.bool,
+    isToggled: PropTypes.bool,
+    type: PropTypes.oneOf(['button', 'reset', 'submit']),
+  };
+
+  render() {
+    return (
+      <button
+        id={this.props.id}
+        name={this.props.name}
+        disabled={this.props.isDisabled}
+        onClick={this.props.onClick}
+        type={this.props.type}
+        ref={this.props.ref}
+        className={styles.reset}
+        // Accessibility
+        role="button"
+        aria-label={this.props.ariaLabel}
```

Comment:
I would make it so that our buttons accept all `aria` props and forward them by default, just like we do for `data-`.

Reply:
not sure I understand. Would you be able to provide a code example of exactly what you mean here?